Makefile

```diff
@@ -125,14 +125,20 @@ endif
 ENABLE_JIT ?= 0
 $(call set-feature, JIT)
 ifeq ($(call has, JIT), 1)
-OBJS_EXT += jit.o
+OBJS_EXT += jit.o t2jit.o
```

Comment:
All tired-2 JIT compiler (T2C) related files should have the prefix `t2c`. That is, `t2c.h`, `t2c.c`, and `t2c_template.c`.